extensions/interactions/TextInput/directives/oppia-interactive-text-input.component.ts

```diff
@@ -33,6 +33,7 @@ interface TextInputSchema {
   'ui_config': {
     placeholder?: string;
     rows?: number;
+    catchMisspellings?: boolean;
```

Comment:
Why is it optional (`?`)?

Reply:
It appeared as an optional parameter [here](https://github.com/oppia/oppia/blob/95f5f83b5a17206072cb76590bc6df87c70f2f9a/extensions/interactions/TextInput/directives/oppia-interactive-text-input.component.ts#L97), hence, the optional part.

Reply:
Ya, apart from the optional use in ui_config, it has a default value of false so, I think keeping it optional works fine.

Reply:
I don't understand what you mean.

Reply:
I'm not sure I get the question. What I meant was that since it is an optional parameter in the ui_config property, we need to keep it optional [here](https://github.com/oppia/oppia/blob/95f5f83b5a17206072cb76590bc6df87c70f2f9a/extensions/interactions/TextInput/directives/oppia-interactive-text-input.component.ts#L97).

Reply:
I'm asking whether you can investigate whether the param in ui_config needs to be optional?

Reply:
I think it's not necessary. We could also assign default values it the fields of ui_config.

Reply:
I think it's not necessary. We could also assign default values it the fields of ui_config.

Reply:
Can you fix that then?

Reply:
Done. PTAL @vojtechjelinek!